tests/test_posixtest.py

```diff
@@ -135,9 +135,16 @@ def get_pthread_tests():
   'test_pthread_spin_lock_3_1': 'signals are not supported',
 }
 
+# Mark certain tests as flaky, which may sometimes fail.
+# TODO invesigate these tests.
+flaky = {
+  'test_pthread_cond_signal_1_1': 'test has tight timing requirements, may fail when test suite is run in parallel',
```

Comment:
In fact if you just make the string here "flaky: <bug_url>" you can put the rest of the details in the bug report rather than here.

Reply:
Ok, just to make things clear to me. You mean that I should file a bug as described here https://emscripten.org/docs/getting_started/bug_reports.html#bug-reports and then use URL of this newly reported bug? Or should I report this bug somewhere else?